containers/server-image/Dockerfile

```diff
@@ -0,0 +1,67 @@
+# SPDX-License-Identifier: MIT
+#!BuildTag: uyuni/server:latest uyuni/server:4.4.0-1 uyuni/server:4.4.0-1.%RELEASE%
+
+ARG BASE=registry.suse.com/bci/bci-base:15.4
+FROM $BASE
+
+# TODO consider setting product repos from the configuration
+ARG PRODUCT_REPO
+
+# Add distro and product repos
+COPY add_repos.sh /usr/bin
+RUN sh add_repos.sh ${PRODUCT_REPO}
+
+# Main packages
+RUN zypper ref
+RUN zypper -n in vi
+RUN zypper -n in python3-simplejson
+RUN zypper --gpg-auto-import-keys --non-interactive install --auto-agree-with-licenses  \
+    patterns-uyuni_server
+RUN zypper mr -d repo-backports-update repo-sle-update
+
+# Additional configuration
+COPY configure.sh /usr/bin
+RUN sh configure.sh
+
+# LABELs
```

Comment:
we are missing the "copy" of the setup_env.sh, which can be useful to manually set up the server